src/library/check_vsd_license_validity.py

```diff
@@ -66,9 +72,19 @@ def check_licenses_expiration(licenses, required_days_left):
                                 days_left)
 
 
-def get_vsd_session(vsd_auth):
-    # Format api version
-    version = 'v5_0'
+def format_api_version(version):
```

Comment:
I think we should handle this differently. First, the 'v3_2' return is for a version we don't support. The check for startswith('3') can be removed. Second, the ('v' + version[0] = '_0') return is for v4_0 and v5_0, of which we only support v5_0. Third, when the release becomes 21.* we will positively fail by returning '21_0' - which won't be correct at all.

I think we can simplify this (and other identical code) to be something like this:

```
if version.startswith('5'):
    return ('v5_0')
else:
    return ('v6')
```
I think this will even work when we go to 21 and rev the API to 7. Not sure.